src/function-configuration.ts

```diff
@@ -72,11 +74,11 @@ export const INGRESS_SETTINGS_OPTIONS = [
  * Scheduler retry options. Applies only to scheduled functions.
  */
 export interface ScheduleRetryConfig {
-  retryCount?: number;
-  maxRetryDuration?: string;
-  minBackoffDuration?: string;
-  maxBackoffDuration?: string;
-  maxDoublings?: number;
+  retryCount?: number | Expression<number> | null;
```

Comment:
No. I want to err on the side of readability. If a user looks up `retryCount` and sees `number | Expression<number> | null` they probably have a better understanding of the world from this API than if they saw `Field<number>` because in the `Field` case, they _must_ look up another page of documentation.

Reply:
Ah that's a good point.